.travis.yml

```diff
@@ -2,10 +2,14 @@ language: ruby
 rvm:
   - 2.2.3
 before_script:
+  # Markdown Linter using custom rule style file.
+  - mdl README.md --style _linter/markdown-linter-style.rb
+  - mdl _posts --style _linter/markdown-linter-style.rb
+  - mdl _pages --style _linter/markdown-linter-style.rb
```

Comment:
> Per PR description, yes, we could simplify those 3 commands down to 1 command on ./ however, the linter did find issues in the m-lab observatory submodule's README. If we updated the submodule's README to pass the linting rules then we can change these commands. I didn't see an easy way to just ignore that external submodule's directory, so that is why I opted for the 3 commands.

Ah, bummer. Other two ideas:

1. Does `--git-recurse` recurse into git submodules? If not, then that would work.
1. Can we specify multiple targets per command (e.g. `mdl README.md _posts _pages --style _linter/markdown-linter-style.rb`)?